examples/pytorch/ogb/seal_ogbl/main.py

```diff
@@ -78,17 +85,24 @@ def cached_name(self):
         return 'SEAL_{}_data_{}.pt'.format(self.split, self.percent)
 
     def process(self):
-        examples = []
+        g_list, labels = [], []
         self.dynamic = True
         for i in range(len(self)):
-            examples.append(self[i])
+            g, z, x, weights, y = self[i]
+            g.ndata['z'] = z
+            if x is not None:
+                g.ndata['x'] = x
+            if weights is not None:
+                g.edata['w'] = weights
```

Comment:
Are `z, x, weights` retrieved or popped from `g`? If it's the former, then there should be no need to reassign them to the graph.